src/components/icons/addToCartIcon.tsx

```diff
@@ -0,0 +1,18 @@
+export const AddToCartIcon = () => {
```

Comment:
To są te ikony `svg`, które wcześniej były bezpośrednio w komponencie, a teraz są importowane jako komponent.